tests/UI/pages/BO/advancedParameters/email/index.js

```diff
@@ -194,6 +203,39 @@ class Email extends BOBasePage {
     return this.getAlertSuccessBlockParagraphContent(page);
   }
 
+  /** setup the smtp parameters
+   *
+   * @returns {Promise<void>}
+   */
+  async setupSmtpParameters(page, server, username, pass, port, encryption = 'None') {
+    // Click on smtp radio button
+    await page.click(this.smtpParametersRadioButton);
+    await this.waitForVisibleSelector(page, this.smtpServerFormField);
+    // fill the form field
+    await this.setValue(page, this.smtpServerFormField, server);
+    await this.setValue(page, this.smtpUsernameFormField, username);
+    await this.setValue(page, this.smtpPasswordFormField, pass);
+    await this.setValue(page, this.smtpPortFormField, port);
+    await this.selectByVisibleText(page, this.smtpEncryptionFormField, encryption);
+    // Click on Save button
+    await this.clickAndWaitForNavigation(page, this.saveEmailFormButton);
+    return this.getAlertSuccessBlockParagraphContent(page);
+  }
+
+  /**
+   * Reset the mail parameters to default config
+   * @param page
+   * @returns {Promise<string>}
+   */
+  async resetDefaultParameters(page) {
+    // Click on smtp radio button
+    await page.click(this.sendMailParametersRadioButton);
+    await this.elementNotVisible(page, this.smtpServerFormField);
```

Comment:
```suggestion
    await page.waitForSelector(this.smtpServerFormField, {state: hidden});
```
The function `elementNotVisible` does not throw exception if the element is not visible.